e2e/backend_service_update_test.go

```diff
@@ -0,0 +1,200 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/exec"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/rnode"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/rnode/backendservice"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/rnode/tcproute"
+	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/rgraph/workflow/plan"
+	"github.com/kr/pretty"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/networkservices/v1"
+)
+
+func buildTcpRoute(builder *rgraph.Builder, name string, serviceName string) (*cloud.ResourceID, error) {
```

Comment:
Can we reuse `buildTCPRoute` from `e2e/tcproute_test.go` or at least rename one of them? There are two differences - `Matches` and `Meshes` fields.